### PR TITLE
Image Component

### PR DESCRIPTION
**Pull request by arpit01923**
- I have made an image component in which

1. Two images variations are made - Round and square
2. I have made it responsive with respect to mobile, tablet and desktop
image
image
image

### CODE REVIEW

File: css/example/Image.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Image</title>
+    <link rel="stylesheet" href="../components/Image.css">
+</head>
+<body>
+
+    <section class="image-round">
+            <img class="size-xl round" src="https://images.unsplash.com/photo-1643188441507-9b8ccc72f755?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDI3fDZzTVZqVExTa2VRfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
+            <img class="size-xl round" src="https://images.unsplash.com/photo-1643620772691-e9d89e73a292?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDMxfDZzTVZqVExTa2VRfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60" alt="">
```
Comment (JaanviNL) [inline css/example/Image.html]: Try not to use the direct url of image. Instead save the image in a separate folder/file and import it in your file.

**Comment by JaanviNL** [comment]
Rest all seems Good.

**Review by JaanviNL — APPROVED** [review]

File: css/example/Image.html (hunk 1 above)
Comment (arpit01923) [reply css/example/Image.html]: It's just a demo component, I will make it easy for reviewers to review the code.
I will make folders/ file and import stuff in future

File: css/components/Image.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+.retina{
+    border-radius: var(--retina-border radius);
+}
+
+@media screen and (max-width: 480px){
+    .size-xl{
+        width: 7rem;
+        height: 7rem;
+        margin: 1rem;
```
Comment (bajajcodes) [inline css/components/Image.css:26]: This line of code is repetitve aka no need to write again here.

File: css/components/Image.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+@media screen and (min-width: 480px) and (max-width: 768px){
+    .size-xl{
+        width: 9rem;
+        height: 9rem;
+        margin: 1.5rem;
+    }    
+}
+
+@media screen and (min-width: 768px){
```
Comment (bajajcodes) [inline css/components/Image.css]: Why you have applied style on this breakpoint?

**Review by bajajcodes — COMMENTED** [review]
If possible read about max-width property and add comments describing your code.

**Review by HarshithaKonduru214 — APPROVED** [review]

File: css/components/Image.css (hunk 2 above)
Comment (arpit01923) [reply css/components/Image.css:26]: conflict resolved

File: css/components/Image.css (hunk 3 above)
Comment (arpit01923) [reply css/components/Image.css]: Conflict resolved

**Pull request merged**